project/src/main/webapp/index.html

```diff
@@ -100,18 +100,22 @@ <h2>Food Mood's recommendation:</h2>
         </div>
         <div id="results-map-container"> </div>
       </div>
-      <div id="feedback-box">
+      <div id="feedback-box" style="display: none">
         <label for="liking-feedback">Like these options?<br> Let us know what you chose: <br></label>
-        <select class="button" name="feedback" id="feedback">
-          <option value="1">First option</option>
-          <option value="2">Second option</option>
-          <option value="3">Third option</option>
-          <br>
-          <input type="submit" value="Submit" style="font-weight: bold">
-        </select>
+        <form id="chosen-place-form" class="form-element-toolbar">
+          <input type="radio" id="option-1" value="1" checked="checked">
```

Comment:
Why is this checked already? It can make this be set as chosen while the user didn't actively choose this.
Also, just wondering - why did you change to radio rather then the drop down we had?

Reply:
Radio - only because that's the way I know how to read the chosen property in the javascript. if you prefer it as a scroll list I think it's possible, just would take a little time to make sure we take the information correctly, I preffered to spare the time.
It's checked for convenience.. not a critical point but I think it's nicer that something is already checked. either way sending the feedback is an active action, because the user has to press "submit"

Reply:
Ok I agree we don't have the time for these front end stuff